src/vizier/services/adaptive_export/internal/sink/clickhouse_test.go

```
@@ -0,0 +1,181 @@
+		TStart:     t0.Add(-5 * time.Minute),
+		TEnd:       t0.Add(5 * time.Minute),
+		LastSeen:   t0,
+		LastRuleID: "R1005",
+		NAnomalies: 1,
+	}
+}
+
+// TestSink_Write_PostsCorrectQueryAndBody — INSERT targets the right
+// table; body is one JSON object per line with all attribution fields.
+func TestSink_Write_PostsCorrectQueryAndBody(t *testing.T) {
+	var (
```

golangci-lint (2.12.1)

[error] File is not properly formatted

(gofumpt)